app/src/main/java/de/westnordost/streetcomplete/quests/camping/AddCampType.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.camping
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddCampType : OsmFilterQuestType<CampType>() {
+
+    override val elementFilter = """
+        nodes, ways with (
+          tourism=camp_site
+        )
+        and (!caravans or !tents)
+        and !backcountry
+    """
+    override val changesetComment = "Survey who may camp here"
+    override val wikiLink = "Key:caravans"
+    override val icon = R.drawable.ic_quest_camp_type
+    override val defaultDisabledMessage = R.string.default_disabled_msg_go_inside
```

Comment:
Maybe the `AddCampType` quest doesn't need to be disabled by default, since you can often see caravans and/or tents from the outside?

Reply:
Also shouldn't it have a slightly different message because you don't have to go "inside" somewhere to check it?

Reply:
ok, I've made it enabled by default with comment in 270a6947c. We'll see if it is very disliked at final review.

@peternewman I think `default_disabled_msg_go_inside` is good enough IMHO; I don't think there is no need to introduce special new disabled categories with just a minor difference in meaning.

Reply:
> ok, I've made it enabled by default with comment in [270a694](https://github.com/streetcomplete/StreetComplete/commit/270a6947c5f9c7f33fce73ab42546ebc42fa3914). We'll see if it is very disliked at final review.

I (and @matkoniecz ) aren't entirely convinced by that.

> @peternewman I think `default_disabled_msg_go_inside` is good enough IMHO; I don't think there is no need to introduce special new disabled categories with just a minor difference in meaning.

Yeah as mentioned the message is more general than the string name.